ground/src/main/java/com/google/android/ground/persistence/remote/firebase/schema/LoiConverter.kt

```diff
@@ -42,7 +42,11 @@ object LoiConverter {
     val loiProto = LocationOfInterestProto::class.parseFrom(doc, 1)
     val geometry = loiProto.geometry.toGeometry()
     val jobId = loiProto.jobId
-    val job = DataStoreException.checkNotNull(survey.getJob(jobId), "job $jobId")
+    val job =
+      DataStoreException.checkNotNull(
+        survey.getJob(jobId),
+        "job $jobId \n loiId = $loiId \n loiProto = $loiProto",
+      )
```

Comment:
This still doesn't make it to Crashlytics, it's getting swallowed and written to local logs here:

https://github.com/google/ground-android/blob/d704635e2b4885615472a2423dc26f8b6e6eedf1/ground/src/main/java/com/google/android/ground/persistence/remote/firebase/schema/LoiCollectionReference.kt#L63-L70

Instead of just logging a warning we should send the error to Crashlytics.

Reply:
Example:

https://github.com/google/ground-android/blob/d704635e2b4885615472a2423dc26f8b6e6eedf1/ground/src/main/java/com/google/android/ground/persistence/sync/SurveySyncWorker.kt#L58

Reply:
updated